.github/workflows/golangci-lint.yml

```diff
@@ -0,0 +1,29 @@
+name: golangci-lint
+on:
+  push:
+    paths:
+      - "go.sum"
+      - "go.mod"
+      - "**.go"
+      - "scripts/errcheck_excludes.txt"
+      - ".github/workflows/golangci-lint.yml"
+  pull_request:
+    paths:
+      - "go.sum"
+      - "go.mod"
+      - "**.go"
+      - "scripts/errcheck_excludes.txt"
+      - ".github/workflows/golangci-lint.yml"
+
+jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
+
+      - name: Lint
+        uses: golangci/golangci-lint-action@v2
+        with:
+          version: v1.39.0
```

Comment:
you can upgrade to the last version ` v1.42.0` :)

Reply:
I wasn't sure if we were depending on `v1.39.0` for a specific reason.

Reply:
Ah probably ^^ I have no idea about this. But perhaps it's worth to try to upgrade and see what's happening.

Reply:
Yeah I don't really think it could hurt, but you never know.

Reply:
perhaps @LeviHarrison you could just try to upgrade and see what's happening ?

Reply:
Sure

Reply:
thanks :)

Reply:
Seems to work fine

Reply:
nice !